Makefile

```diff
@@ -82,7 +82,10 @@ LOG_INFO := loginfo
 
 MMU := mmu
 
-MMIO := mmio/columns.lisp #TODO enable the MMIO constraint and lookup
+MMIO := mmio/columns.lisp
+#TODO enable the MMIO constraint and lookup
```

Comment:
why did you change the std way ?

Reply:
You mean why did I move the comment ? I don't know